src/api/Nhs.Appointments.Api/Notifications/template-bookingreminder-rsv-email.txt

```diff
@@ -22,6 +22,7 @@ Date and time: ((date))
 
 # If you need to change or cancel your appointment:
 
-* visit https://www.nhs.uk/book-rsv
+* Visit https://www.nhs.uk/book-rsv
+* If you cant get online, call 119
```

Comment:
We're missing an apostrophe in can't - could make it cannot if the apostrophe causes serialisation issues?

Reply:
oof!